rhino/src/main/java/org/mozilla/javascript/CompoundOperationMap.java

```diff
@@ -0,0 +1,109 @@
+package org.mozilla.javascript;
+
+import java.util.Iterator;
+
+// Should be named something like compound operations map.
```

Comment:
This is a really important bit of new functionality. Can you please add actual Javadoc here so that people can understand now to apply it for future code?